components/gitops/staging/kustomization.yaml

```diff
@@ -2,80 +2,13 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- allow-argocd-to-manage.yaml
-- argo-cd.yaml
+- https://github.com/redhat-appstudio/managed-gitops/manifests/overlays/appstudio-staging-cluster?ref=6197011ff97aa22cefb45e9a96a9022ab8243d2d
```

Comment:
https://github.com/redhat-appstudio/managed-gitops/blob/main/.tekton/push.yaml - should be updated to be able to work with this format, see https://github.com/redhat-appstudio/build-service/blob/main/.tekton/push.yaml#L19 as example

Reply:
This one should be universal -> https://github.com/redhat-appstudio/build-definitions/blob/main/.tekton/push.yaml#L101

Reply:
(should work for image, ref= and when it's inside url)